### PR TITLE
Package name as key in configuration

### PR DESCRIPTION
**Pull request by Nex-Otaku**
| Q             | A
| ------------- | ---
| Is bugfix?    | Yes
| New feature?  | No
| Breaks BC?    | No
| Fixed issues  | #66

### CODE REVIEW

**Comment by vjik** [comment]
Need to check configs in `yiisoft/app` and `yiisoft/yii-demo` for use use changed key.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
But, as @vjik mentioned, it should be adjusted in applications.

**Comment by samdark** [comment]
https://github.com/yiisoft/app/pull/136

File: src/FastRouteCache.php [hunk 1]
```diff
@@ -23,9 +23,9 @@ final class FastRouteCache implements CacheInterface
      * Template used when generating the cache file.
      */
     public const CACHE_TEMPLATE = <<< 'EOT'
-<?php
-return %s;
-EOT;
+    <?php
+    return %s;
+    EOT;
```
Comment (roxblnfk) [inline src/FastRouteCache.php:28]:
```suggestion
    "<?php\nreturn %s;\n";
```
Why not? :D

**Review by roxblnfk — APPROVED** [review]

**Comment by samdark** [comment]
Should be adjusted as well in yii-demo, app-api, demo-api.

**Comment by Nex-Otaku** [comment]
@vjik @samdark 

I am working on it, adjusting to demo and other.

File: src/FastRouteCache.php (hunk 1 above)
Comment (Nex-Otaku) [reply src/FastRouteCache.php:28]: Because of code style consistency.

Comment (roxblnfk) [reply src/FastRouteCache.php:28]: Then
```suggestion
        <?php
        return %s;
        EOT;
```

**Comment by Nex-Otaku** [comment]
Demo PR https://github.com/yiisoft/yii-demo/pull/261

**Comment by Nex-Otaku** [comment]
App PR https://github.com/yiisoft/app/pull/137

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Comment by Nex-Otaku** [comment]
App API PR https://github.com/yiisoft/app-api/pull/53

**Comment by Nex-Otaku** [comment]
Demo API PR https://github.com/yiisoft/demo-api/pull/16

**Comment by samdark** [comment]
Merged both. Thanks.

**Pull request merged**